docker-compose-host-nfs.yml

```diff
@@ -0,0 +1,74 @@
+version: "2.1"
+
+services:
+#  credentials:
```

Comment:
I suppose you'd like to keep the commented out code, right?

Reply:
Yes